tests/Functional/ProductManagerTest.php

```diff
@@ -368,128 +469,13 @@ public function filters(): array
         ];
     }
 
-    public function getResponse(): string
+    public function getResponse($schema = null): string
```

Comment:
it miss define the type of variable it will receive

Reply:
Added parameter type. Thanks!